<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profil</class>
 <widget class="QWidget" name="profil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>501</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#5a9e94;
color:#21421e;
border-stule:outset;
border-width:2px;
border-radius:10px;
border-color:white;
font:bold 14px;
padding:10px;
min-width:10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="deposit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#497e76;
color:#21421e;
border-stule:outset;
border-width:2px;
border-radius:10px;
border-color:white;
font:bold 14px;
padding:10px;
min-width:10px;
</string>
   </property>
   <property name="text">
    <string>Пополнить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bringout">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#497e76;
color:#21421e;
border-stule:outset;
border-width:2px;
border-radius:10px;
border-color:white;
font:bold 14px;
padding:10px;
min-width:10px;
</string>
   </property>
   <property name="text">
    <string>Вывести</string>
   </property>
  </widget>
  <widget class="QPushButton" name="translate">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#497e76;
color:#21421e;
border-stule:outset;
border-width:2px;
border-radius:10px;
border-color:white;
font:bold 14px;
padding:10px;
min-width:10px;
</string>
   </property>
   <property name="text">
    <string>получить код</string>
   </property>
  </widget>
  <widget class="QPushButton" name="convert">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:red;
color:#21421e;
border-stule:outset;
border-width:2px;
border-radius:10px;
border-color:white;
font:bold 10px;
padding:10px;
min-width:10px;
</string>
   </property>
   <property name="text">
    <string>Конвертировать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goout">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>640</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#497e76;
color:#21421e;
border-stule:outset;
border-width:2px;
border-radius:10px;
border-color:white;
font:bold 14px;
padding:10px;
min-width:10px;
</string>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>640</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#497e76;
color:#21421e;
border-stule:outset;
border-width:2px;
border-radius:10px;
border-color:white;
font:bold 14px;
padding:10px;
min-width:10px;
</string>
   </property>
   <property name="text">
    <string>Обновить крус</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-40</y>
     <width>541</width>
     <height>771</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#a3c6c0;
color:#20603d;
border-stule:outset;
border-width:2px;
border-radius:10px;
border-color:white;
font:bold 14px;
padding:10px;
min-width:10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Rubpix">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../image/rub.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;RUB&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;курс&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;кол-во&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="RUB_count">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;число&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Usdtpix">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../image/dol.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;USDT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;курс&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>85</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;кол-во&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="USDT_count">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>173</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;число&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Eurpix">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>331</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../image/eur.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;EUR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;курс&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;кол-во&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="EUR_count">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;число&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Btcpix">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../image/btc.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;BTC&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;курс&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>410</y>
     <width>85</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;кол-во&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="BTC_count">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>410</y>
     <width>173</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;число&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Ethpix">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../image/eth.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;ETH&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;курс&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>470</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;кол-во&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ETH_count">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>84</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;число&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>470</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>label_27</zorder>
  <zorder>label</zorder>
  <zorder>deposit</zorder>
  <zorder>bringout</zorder>
  <zorder>translate</zorder>
  <zorder>convert</zorder>
  <zorder>goout</zorder>
  <zorder>update</zorder>
  <zorder>Rubpix</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
  <zorder>label_4</zorder>
  <zorder>RUB_count</zorder>
  <zorder>Usdtpix</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>USDT_count</zorder>
  <zorder>Eurpix</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_15</zorder>
  <zorder>EUR_count</zorder>
  <zorder>label_14</zorder>
  <zorder>Btcpix</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>label_20</zorder>
  <zorder>BTC_count</zorder>
  <zorder>Ethpix</zorder>
  <zorder>label_22</zorder>
  <zorder>label_23</zorder>
  <zorder>label_25</zorder>
  <zorder>ETH_count</zorder>
  <zorder>label_24</zorder>
  <zorder>label_5</zorder>
  <zorder>label_11</zorder>
  <zorder>label_16</zorder>
  <zorder>label_21</zorder>
  <zorder>label_26</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
